pudl_scrapers/items.py

```diff
@@ -15,13 +15,26 @@ class DataFile(scrapy.Item):
 
 class Eia860(DataFile):
     """The Eia860 forms in a zip file."""
+
     year = scrapy.Field(serializer=int)
 
     def __repr__(self):
         return "Eia860(year=%d, save_path='%s')" % (
             self["year"], self["save_path"])
 
 
+class Eia860M(DataFile):
+    """The Eia860M forms in a zip file."""
```

Comment:
Do they get zipped up? Or are they just Excel files?

Reply:
just excel! everything stays in its native format